multi-currency/client/data/index.ts

```diff
@@ -0,0 +1,20 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from './constants';
+import { initStore } from './store';
+
+initStore();
```

Comment:
As discussed on the call, I tried loading the multi-currency settings page and got an error for the `wc/payments` store already being registered. This is likely due to what this PR is trying to accomplish by staying lean and avoiding merge conflicts. We decided a path forward would be to confirm the progress on the other two ongoing issues and if those are far enough along, we'll merge this PR as is and handle the full updated linking in the relevant PRs. If those issues are not far along, we'll update the linking in this PR (or just one of the others) to give us a better sense of where the gaps are now.

Reply:
Correct. Here are links to the other two PRs for reference:
* #9326 
* #9348 

Those two PRs are far along already, so I will let each PR do their respective linking.